HOWTO_PACKAGE.md

```diff
@@ -11,7 +11,7 @@ a 16.04 Ubuntu machine.
 This is a non-exhaustive list of what you'll need:
 
 ```
-sudo apt-get install pbuilder debootstrap devscripts libnautilus-extension-dev mock rpm expect createrepo
+sudo apt-get install pbuilder debootstrap devscripts libnautilus-extension-dev mock rpm expect createrepo cdbs gnome-common debian-archive-keyring
 ```
 
 3. Copy .pbuilderrc to ~/.pbuilderrc
```

Comment:
It may be worth adding a note to `5` about the error you hit when building the package b/c the user building it wasn't in the `mock` group.

Reply:
Oh, I'll mention `The password you typed is invalid.` error on 5. Thanks!